guides/giving_donations_integration.md

```diff
@@ -1,6 +1,22 @@
-# Giving Donations Integration
+# Building A Giving Integration
 
-Because of the way that the Planning Center apps are structured in relation to each other and how Giving itself is built, there are a few concepts that are particularly helpful to know when building any sort of API integration that creates Donation records in Giving, especially on an ongoing basis.
+So you're looking to build a Giving integration? Great! This guide will give you all the info you need to get up and running. To start with, let's go over the two ways your integration can interact with Giving: the Giving API and Webhooks.
+
+## Giving API
+
+Our API (Application Programming Interface) gives you read and write access to all the essential models (`Donation`s, `Donor`s, `Fund`s, etc.) used in Giving. It allows you to automate tasks that would typically be performed via the web UI or export data from Giving into your own application. Be sure to check out Planning Center's full [API Documentation](https://developer.planning.center/docs/#/introduction).
+
+## Webhooks
+
+In addition to performing actions via the API, your integration may want to keep track of changes made to the Giving database. As of April 2021, Giving offers webhook events so that you can be immediately notified anytime a record is created, updated, or destroyed, and take some action in response. We'll go over webhooks in [more detail down below](#all-about-webhooks), and we also recommend taking a look at Planning Center's full [Webhooks Documentation](https://developer.planning.center/docs/#/apps/webhooks).
+
+## Which should I use?
+
+These tools are meant to be used in tandem by your integration. For example, you may want to offer a complete list of donations for a given Church/Organization. To do this, you would start with a full export of `Donation`s using the API, and then subscribe to `Donation` webhook events to keep in sync with any new donations made. In short, the API should be used to **perform actions** or **view historical data**. Webhooks should be used to **keep your data in sync** with Giving.
+
+# Getting to know Giving data models
+
+Because of the way that Planning Center apps are structured in relation to each other and how Giving itself is built, there are a few concepts that are particularly helpful to know when building any sort of API integration that creates Donation records in Giving, especially on an ongoing basis.
```

Comment:
Although we still say "apps" internally sometimes, we try and use the term "products" when referring to the main web products in outward-facing documentation and in support.